packages/keystore_ffi/android/src/main/jniLibs/arm64-v8a/libkeystore_ffi.so

```diff
@@ -0,0 +1 @@
+/Users/shadykhalifa/Code/sunshine-flutter/target/aarch64-linux-android/debug/libkeystore_ffi.so
```

Comment:
please use relative symlinks

Reply:
this just a stub file, it will get overwritten when you run `cargo make`